### PR TITLE
add(epp): protocol sftp

### PR DESCRIPTION
**Pull request by garnier-quentin**
## Target version

- [ ] 20.10.x (staging)
- [ ] 21.04.x (staging)
- [ ] 21.10.x (staging)
- [ ] Cloud (staging)
- [X] Plugin Packs (staging)
- [ ] 22.04.x (next)

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-sftp.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+| step status                                   | Step status              |       |    
+| *step_label*#step.execution.time.milliseconds | Step execution time      | ms    |
+
+</TabItem>
+</Tabs>
+
+## Prérequis
+
+Pour superviser votre serveur SFTP, il est possible de réaliser une authentification par : 
+* clés publique
```
Comment (lchrdn) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-sftp.md:66]:
```suggestion
* clé publique
```

**Review by lchrdn — APPROVED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-sftp.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+title: Protocol SFTP
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Contenu du Pack
+
+### Modèles
+
+Le Pack Centreon SFTP apporte 1 modèle d'hôte :
```
Comment (Sims24) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-sftp.md:12]:
```suggestion
Le Pack Centreon SFTP apporte un modèle d'hôte :
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-sftp.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+* App-Protocol-Sftp-custom
+
+Il apporte les Modèles de Service suivants :
+
+| Service Alias  | Service Templat               | Default | Discovery |
```
Comment (Sims24) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-sftp.md:17]: s/Service Alias/Alias/
s/Service Templat/Modèle de services/
s/Default/Défaut/
s/Discovery/Découverte/

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-sftp.md [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+Pour superviser votre serveur SFTP, il est possible de réaliser une authentification par : 
+* clé publique
+* utilisateur et mot de passe
+
+## Installation
+
+<Tabs groupId="sync">
+<TabItem value="Online License" label="Online License">
+
+1. Installer le Plugin Centreon sur tous les collecteurs Centreon devant superviser des ressources **Protocol SFTP** :
```
Comment (Sims24) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-sftp.md:74]:
```suggestion
1. Installer le plugin Centreon sur tous les collecteurs Centreon devant superviser des ressources **Protocol SFTP** :
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-sftp.md [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+
+```bash
+yum install centreon-plugin-Applications-Protocol-Sftp
+```
+
+2. Sur l'interface Web de Centreon, installer le Pack **Protocol SFTP** depuis la page **Configuration > Packs de plugins**.
```
Comment (Sims24) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-sftp.md:80]:
```suggestion
2. Sur l'interface web de Centreon, installer le Pack **Protocol SFTP** depuis la page **Configuration > Packs de plugins**.
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-sftp.md [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+
+</TabItem>
+
+<TabItem value="Offline License" label="Offline License">
+
+1. Installer le Plugin Centreon sur tous les collecteurs Centreon devant superviser des ressources **Protocol SFTP** :
```
Comment (Sims24) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-sftp.md:86]:
```suggestion
1. Installer le plugin Centreon sur tous les collecteurs Centreon devant superviser des ressources **Protocol SFTP** :
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-sftp.md [hunk 7]
```diff
@@ -0,0 +1,164 @@
...
+
+```bash
+yum install centreon-plugin-Applications-Protocol-Sftp
+```
+
+2. Sur le serveur Central Centreon, installer le RPM du Pack **Protocol SFTP** :
```
Comment (Sims24) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-sftp.md:92]:
```suggestion
2. Sur le serveur central Centreon, installer le RPM du Pack **Protocol SFTP** :
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-sftp.md [hunk 8]
```diff
@@ -0,0 +1,164 @@
...
+
+```bash
+yum install centreon-pack-applications-protocol-sftp
+```
+
+3. Sur l'interface Web de Centreon, installer le Pack **Protocol SFTP** depuis la page **Configuration > Packs de plugins**.
```
Comment (Sims24) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-sftp.md:98]:
```suggestion
3. Sur l'interface web de Centreon, installer le Pack **Protocol SFTP** depuis la page **Configuration > Packs de plugins**.
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-sftp.md [hunk 9]
```diff
@@ -0,0 +1,164 @@
...
+
+</TabItem>
+</Tabs>
+
+## Configuration
+
+### Hôte
+
+* Ajoutez un Hôte à Centreon depuis la page **Configuration > Hôtes**
```
Comment (Sims24) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-sftp.md:107]:
```suggestion
* Ajoutez un hôte à Centreon depuis la page **Configuration > Hôtes**
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-sftp.md [hunk 10]
```diff
@@ -0,0 +1,164 @@
...
+* Complétez les champs **Nom**, **Alias** & **IP Address / DNS** correspondant à votre serveur **Protocol SFTP**.
+* Appliquez le modèle d'hôte **App-Protocol-Sftp-custom**.
+* Une fois le modèle appliqué, les macros ci-dessous indiquées comme requises (**Obligatoire**) doivent être renseignées.
+
+Une fois celui-ci configuré, certaines macros doivent être renseignées:
+
+| Mandatory | Name                     | Description                                                                |
```
Comment (Sims24) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-sftp.md:114]: s/Mandatory/Obligatoire/
s/Name/Nom/

File: pp/integrations/plugin-packs/procedures/applications-protocol-sftp.md [hunk 11]
```diff
@@ -0,0 +1,160 @@
...
+title: Protocol SFTP
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Pack Assets
+
+### Templates
+
+The Centreon Pack SFTP brings 1 host template:
```
Comment (Sims24) [inline pp/integrations/plugin-packs/procedures/applications-protocol-sftp.md:12]:
```suggestion
The Centreon Pack SFTP brings a host template:
```

File: pp/integrations/plugin-packs/procedures/applications-protocol-sftp.md [hunk 12]
```diff
@@ -0,0 +1,160 @@
...
+## Prerequisites
+
+To monitor your SFTP server, you can use pubkey authentication and/or username authentication.
+
+## Setup
+
+<Tabs groupId="sync">
+<TabItem value="Online License" label="Online License">
+
+1. Install the Centreon Plugin package on every Centreon poller expected to monitor **Protocol SFTP** resources:
```
Comment (Sims24) [inline pp/integrations/plugin-packs/procedures/applications-protocol-sftp.md:72]:
```suggestion
1. Install the Centreon plugin package on every Centreon poller expected to monitor **Protocol SFTP** resources:
```

File: pp/integrations/plugin-packs/procedures/applications-protocol-sftp.md [hunk 13]
```diff
@@ -0,0 +1,160 @@
...
+
+```bash
+yum install centreon-plugin-Applications-Protocol-Sftp
+```
+
+2. On the Centreon Web interface, install the **Protocol SFTP** Centreon Pack on the **Configuration > Plugin Packs** page.
```
Comment (Sims24) [inline pp/integrations/plugin-packs/procedures/applications-protocol-sftp.md:78]:
```suggestion
2. On the Centreon web interface, install the **Protocol SFTP** Centreon Pack on the **Configuration > Plugin Packs** page.
```

File: pp/integrations/plugin-packs/procedures/applications-protocol-sftp.md [hunk 14]
```diff
@@ -0,0 +1,160 @@
...
+
+</TabItem>
+
+<TabItem value="Offline License" label="Offline License">
+
+1. Install the Centreon Plugin package on every Centreon poller expected to monitor **Protocol SFTP** resources:
```
Comment (Sims24) [inline pp/integrations/plugin-packs/procedures/applications-protocol-sftp.md:84]:
```suggestion
1. Install the Centreon plugin package on every Centreon poller expected to monitor **Protocol SFTP** resources:
```

File: pp/integrations/plugin-packs/procedures/applications-protocol-sftp.md [hunk 15]
```diff
@@ -0,0 +1,160 @@
...
+yum install centreon-plugin-Applications-Protocol-Sftp
+```
+
+2. Install the **Protocol SFTP** Centreon Pack RPM on the Centreon Central server:
+
+```bash
+yum install centreon-pack-applications-protocol-sftp
+```
+
+3. On the Centreon Web interface, install the **Protocol SFTP** Centreon Pack on the **Configuration > Plugin Packs** page.
```
Comment (Sims24) [inline pp/integrations/plugin-packs/procedures/applications-protocol-sftp.md:96]:
```suggestion
3. On the Centreon web interface, install the **Protocol SFTP** Centreon Pack on the **Configuration > Plugin Packs** page.
```

File: pp/integrations/plugin-packs/procedures/applications-protocol-sftp.md [hunk 16]
```diff
@@ -0,0 +1,160 @@
...
+
+</TabItem>
+</Tabs>
+
+## Configuration
+
+### Host
+
+* Log into Centreon and add a new Host through **Configuration > Hosts**.
```
Comment (Sims24) [inline pp/integrations/plugin-packs/procedures/applications-protocol-sftp.md:105]:
```suggestion
* Log into Centreon and add a new host through **Configuration > Hosts**.
```

**Review by Sims24 — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-sftp.md [hunk 17]
```diff
@@ -0,0 +1,164 @@
...
+import TabItem from '@theme/TabItem';
+
+## Contenu du Pack
+
+### Modèles
+
+Le Pack Centreon SFTP apporte un modèle d'hôte :
+* App-Protocol-Sftp-custom
+
+Il apporte les Modèles de Service suivants :
```
Comment (lchrdn) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-sftp.md:15]:
```suggestion
Il apporte les modèles de service suivants :
```
```suggestion
Il apporte les Modèles de Service suivants :
```

Comment (lchrdn) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-sftp.md:15]:
```suggestion
Il apporte les modèles de service suivants :
```
```suggestion
Il apporte les Modèles de Service suivants :
```

**Review by lchrdn — APPROVED** [review]

**Review by Sims24 — APPROVED** [review]

**Pull request merged**